packages/block-library/src/navigation/edit/unsaved-inner-blocks.js

```diff
@@ -170,17 +158,5 @@ export default function UnsavedInnerBlocks( {
 
 	const Wrapper = isSaving ? Disabled : 'div';
 
-	return (
-		<>
-			{ isSaving ? (
-				<Spinner
-					className={
-						'wp-block-navigation__uncontrolled-inner-blocks-loading-indicator'
-					}
-				/>
-			) : (
-				<Wrapper { ...innerBlocksProps } />
-			) }
```

Comment:
We no longer require this logic as the component will not be rendered when creating a menu.